qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/ClusteringRule.java

```diff
@@ -348,7 +348,9 @@ private Broker createBroker(final int nodeId) {
             new BrokerClusterConfiguration().clusterConfig(brokerCfg),
             Version.from(VersionUtil.getVersion()));
     final var scheduler =
-        new ActorSchedulerConfiguration(brokerCfg, actorClockConfiguration).scheduler();
+        new io.camunda.zeebe.broker.ActorSchedulerConfiguration(
```

Comment:
💭 🔧 Hm couldn't we have one?

Reply:
I don't understand the comment :sweat_smile:

Reply:
Sorry. One configuration instead of two.

Reply:
Ah, because this refers to the Spring `@Configuration` component :smile: It's not a "static" config in that sense, but a component which produces some beans, and the gateway and broker produce different `ActorScheduler` bean :sweat_smile: (e.g. the gateway hard codes I/O threads to 0).